index.html

```diff
@@ -2031,17 +2031,17 @@ <h2 id="presentations">Presentations</h2>
   "verifiableCredential": [
     {
       "@context": "https://www.w3.org/ns/credentials/v2",
-      "id": "data:application/vc-ld+cose,QzVjV...RMjU",
+      "id": "data:application/vc+cose,QzVjV...RMjU",
```

Comment:
technically you need a base encoding here... since +cose is a binary media type

Reply:
I added a note on using hex for this, please let me know if you feel we need more

Reply:
I'll claim no expertise in this area but I don't think that's how any of it works.

Reply:
yes .. you're right. it's unfortunate the spec is either ASCII or base64. I believe we'll need to change all examples to base64.

Reply:
updated